op-service/eth/types.go

```diff
@@ -261,6 +261,15 @@ func BlockAsPayload(bl *types.Block, canyonForkTime *uint64) (*ExecutionPayload,
 			return nil, fmt.Errorf("tx %d failed to marshal: %w", i, err)
 		}
 		opaqueTxs[i] = otx
+		// [Kroma: START] Use KromaDepositTx instead DepositTx
```

Comment:
Suddenly I had this thought. 
Couldn't we just apply IsKromaDepositTx to encodeTyped paired with the decodeTyped function and not have to put this code in?

Reply:
This code is necessary because DepositTx should only be applied after MPT time.